src/main/resources/jenkins/plugins/publish_over_ssh/BapSshPublisher/config.jelly

```diff
@@ -66,7 +66,7 @@
         <j:scope>
             <j:set var="descriptor" value="${descriptor.transferDescriptor}"/>
             <f:repeatable items="${instance.transfers}" var="instance" name="transfers" minimum="1" header="${m.transfers_dragAndDropLabel()}" add="${m.addTransfer()}">
-                <s:blockWrapper> width="100%" padding="0" class="ssh-transfer">
```

Comment:
Does this not imply a corresponding change to the logic that deals with `ssh-transfer` in `src/main/webapp/js/pos.js`?